src/core/mac/mac_frame.hpp

```diff
@@ -168,6 +191,17 @@ class ExtAddress : public otExtAddress
      */
     void ToggleLocal(void) { m8[0] ^= kLocalFlag; }
 
+    /**
+     * This method copies the Extended Address into a given buffer.
+     *
+     * @param[out]  A pointer to buffer to copy the Extended Address into in reverse order.
```

Comment:
nit: whether or not `in reverse order` may depend on `aByteOrder`?
missing doxgen for `aByteOrder`

Reply:
Thanks. Fixed them in new pushed commit.